src/templates/base/store/global/context.jsx

```diff
@@ -0,0 +1,16 @@
+// @context.jsx 
+// will be a provider for our context
+
+import React, { useReducer } from 'react';
+import reducer, {initialState} from './reducer';
+
+// Creating global context
+export const context = React.createContext();
+const {Provider} = {...context};
```

Comment:
No real need for this deconstruct as we only use it once so better would be  

<Context.Provider value={state}>{props.children}</Context.Provider>

Reply:
done